docs/config/usage.md

```diff
@@ -357,9 +357,9 @@ approach are:
 mermaid.startOnLoad = true;
 ```
 
-```warning
+::: warning
```

Comment:
actually vitepress need those to be ::: by default. 
did you meant to turn ::: in ``` in the transform step?
then it would be better to transform in 
```
> **Note**
> This is a note

> **Warning**
> This is a warning
```

that renders as:
> **Note**
> This is a note

> **Warning**
> This is a warning

Reply:
done if it is ok with you

Reply:
> actually vitepress need those to be ::: by default.

This is the flow I had in mind. Is this not how it's done?

```mermaid
flowchart
src["/packages/mermaid/src/docs"] --> |Transform| docs["/docs"] --> |Vitepress| gh[GitHub pages]
docs --> g[Github Repo]
``` 

Is this how it's being done now?
```mermaid
flowchart
src["/packages/mermaid/src/docs"] --> |Transform| docs["/docs"] 
src--> |Vitepress| gh[GitHub pages]
docs --> g[Github Repo]
```

I would prefer the source docs to be proper markdown and not have any vitepress specific things.

Reply:
Is the second one.. 

```warning doe not meat "warning" in github
if you remove the warning it is the same.. 
to do what you want we sould use 
```
> **Warning**
> Any text of warning block
```
we can try to do that later and change both transform but it is harder then the current version.